spec/services/document_capture_session_result_spec.rb

```diff
@@ -47,5 +51,66 @@
         expect(result.selfie_status).to be_an_instance_of(Symbol)
       end
     end
+
+    describe '#success?' do
```

Comment:
Do you think it would be worth testing this method's behavior for a scenario where `doc_auth_success` and `selfie_status` don't exist, but the `success` field does?  

I ask because part of this ticket's goal is "to avoid any errors that result from an implementation that attempts to read from a session field that does not exist."  But since we write to the fields `success`, `doc_auth_success`, and `selfie_status` in both the methods `store_failed_auth_data` and `store_result_from_response` in DocumentCaptureSession, I can't think of a scenario where the `success` field would exist and the other fields wouldn't.

Reply:
@eileen-nava , added test for those fields to be `nil`

Reply:
Thank you! 🙏🏻